openwisp_radius/tests/test_api.py

```diff
@@ -2414,3 +2414,63 @@ def test_change_phone_number_restriction(self):
             self.assertEqual(user.phone_number, phone_number)
 
         app_settings.ALLOWED_MOBILE_PREFIXES = []
+
+
+class TestApiRejectPhoneToken(ApiTokenMixin, BaseTestCase):
+
+    _test_email = 'test@openwisp.org'
+
+    def setUp(self):
+        super().setUp()
+        radius_settings = self.default_org.radius_settings
+        radius_settings.sms_verification = False
+        radius_settings.full_clean()
+        radius_settings.save()
+
+    def test_register_201(self):
```

Comment:
Please change the name of the test or it will be confusing.

A good name in this case would be `test_register_201_phone_number_empty`, which indicates the API request will be successful but we expect the phone number to be empty